Compfiles/Imo1970P6.lean

```diff
@@ -30,17 +30,47 @@ def AcuteTriangle (T : Affine.Triangle ℝ Pt) : Prop :=
   ∠ (T.points 2) (T.points 3) (T.points 1) < Real.pi / 2 ∧
   ∠ (T.points 3) (T.points 1) (T.points 2) < Real.pi / 2
 
+
+/-
+Solved by Goedel-Prover-V2: https://arxiv.org/abs/2508.03613
```

Comment:
This comment pertains only to the solution, not the problem statement. So that it does not show up on the problem statement page https://dwrensha.github.io/compfiles/problems/Compfiles.Imo1970P6.html, please put it within the solution tactic block, after the `by`.